package.json

```diff
@@ -7,6 +7,17 @@
     "build": "next build",
     "start": "next start"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*": [
+      "prettier --check . --write",
+      "git add"
```

Comment:
I believe you should remove the `git add` step. I was looking around and found this:


> Prior to version 10, tasks had to manually include `git add` as the final step. This behavior has been integrated into lint-staged itself in order to prevent race conditions with multiple tasks editing the same files. If lint-staged detects `git add` in task configurations, it will show a warning in the console. Please remove `git add` from your configuration after upgrading.

https://github.com/okonet/lint-staged#reformatting-the-code

Reply:
Also after reading the documentation I get to know the `--check` flag is redundant here.